openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx

```diff
@@ -70,6 +70,14 @@ const TourPage = () => {
     setExplorePageCounts(exploreCount);
   };
 
+  /**
+   *
+   * @returns void
+   */
+  const handleFilterChange = () => {
```

Comment:
Why do we need this?

Reply:
Just an empty callback handler for explore component as it's not in use for tours page, but this prop is mandatory